src/test/features/environment-spec.ts

```diff
@@ -25,8 +25,8 @@ import {BasicSpecTemplate} from '../templates.js';
 
 const expect = chai.expect;
 const ALT_BG_IMAGE_URL = assetPath('quick_4k.png');
-const BG_IMAGE_URL = assetPath('spruit_sunrise_2k.jpg');
-const HDR_BG_IMAGE_URL = assetPath('spruit_sunrise_2k.hdr');
+const BG_IMAGE_URL = assetPath('spruit_sunrise_2k_LDR.jpg');
+const HDR_BG_IMAGE_URL = assetPath('spruit_sunrise_2k_HDR.hdr');
```

Comment:
I did this because the caching is done by filename sans extension. Plus it's really easy to overlook the extension and think they're the same file otherwise.